Formula/ansible.rb

```diff
@@ -59,40 +60,24 @@ class Ansible < Formula
   end
   ### end
 
-  resource "Babel" do
```

Comment:
Well, whatever poet does is the format these should have, except for the ones that call out that they need to be at the top. Sorry if I'm sending mixed signals.